### PR TITLE
Make MultiSmelter able to do more than 128 parallels

### PR DESCRIPTION
**Pull request by BlueWeabo**
The multismelter was before only able to do 128 and it got to that with HSS-G coils.

With this change it will be able to go above that  with the higher tier coils.

This change won't cause any more lag than it is currently creating.

### CODE REVIEW

File: src/main/java/gregtech/api/enums/HeatingCoilLevel.java [hunk 1]
```diff
@@ -41,8 +41,8 @@ public byte getTier() {
         /**
          * @return the coil Level, used for Parallels in the Multi Furnace for example.
          */
-        public byte getLevel() {
```
Comment (boubou19) [inline src/main/java/gregtech/api/enums/HeatingCoilLevel.java:44]: you shouldn't change the signature of a public method in the API, unless you went through every single GT adddon and made sure this doesn't introduce any bug.

**Review by boubou19 — CHANGES_REQUESTED** [review]

**Comment by BlueWeabo** [comment]
This should have fixed it and now it only gives UHV+ coils more parallels as that is where it gets to be a problem. (totally not because I got annoyed at the end)

**Comment by Dream-Master** [comment]
@BlueWeabo please resolve the conflict

**Comment by BlueWeabo** [comment]
I think I did it

**Comment by BlueWeabo** [comment]
Yeah, the conflict was resolved.

**Review by boubou19 — APPROVED** [review]

**Pull request merged**